networks/rpc/http.go

```diff
@@ -269,6 +269,7 @@ func NewHTTPServer(cors []string, vhosts []string, timeouts HTTPTimeouts, srv ht
 	// Wrap the CORS-handler within a host-handler
 	handler := newCorsHandler(srv, cors)
 	handler = newVHostHandler(vhosts, handler)
+	handler = http.TimeoutHandler(handler, timeouts.ExecutionTimeout, "timeout")
```

Comment:
Adding a feature in the handler seems like a weird pattern.
I think we need to refactor it later (TODO list)